scenario/consistency_checker.py

```diff
@@ -326,10 +327,17 @@ def check_storages_consistency(
     return Results(errors, [])
 
 
+def _is_secret_identifier(value: Union[str, int, float, bool]):
+    """Return true iff the value is in the form `secret:{secret id}`."""
```

Comment:
Given this docstring I'd expect the return type to be `bool`.
I'd either adjust the return type to Match and rephrase, or cast the Match to bool and add a return type annotation to match.